Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -721,6 +752,35 @@ class SearchViewController: SiteTableViewController,
             twoLineCell.leftImageView.setFavicon(FaviconImageViewModel(siteURLString: site.url))
             twoLineCell.accessoryView = nil
             cell = twoLineCell
+        case .firefoxSuggestions:
+            let firefoxSuggestion = firefoxSuggestions[indexPath.row]
+            let titleText = String(format: .Search.FirefoxSuggestionTitle, firefoxSuggestion.fullKeyword, firefoxSuggestion.title)
+            twoLineCell.titleLabel.text = titleText
+            if Locale.current.languageCode == "en",
+               let fullKeywordRange = titleText.range(of: firefoxSuggestion.fullKeyword),
+               let searchQueryRange = titleText.range(of: savedQuery, options: [.caseInsensitive], range: fullKeywordRange) {
+                // Bold the remainder of the full keyword in the suggestion
+                // title. Both the search query prefix, and everything after the
+                // full keyword, stays normal weight.
+                let attributedString = titleText.attributedText(
+                    boldIn: searchQueryRange.upperBound..<fullKeywordRange.upperBound,
+                    font: LegacyDynamicFontHelper().DefaultStandardFont
+                )
+                twoLineCell.titleLabel.attributedText = attributedString
+            }
+            if firefoxSuggestion.isSponsored {
+                twoLineCell.descriptionLabel.isHidden = false
+                twoLineCell.descriptionLabel.text = .Search.SponsoredSuggestionDescription
+            } else {
+                twoLineCell.descriptionLabel.isHidden = true
+            }
+            twoLineCell.leftOverlayImageView.image = nil
+            twoLineCell.leftImageView.contentMode = .scaleAspectFit
+            twoLineCell.leftImageView.layer.borderColor = SearchViewControllerUX.IconBorderColor.cgColor
+            twoLineCell.leftImageView.layer.borderWidth = SearchViewControllerUX.IconBorderWidth
+            twoLineCell.leftImageView.image = firefoxSuggestion.iconImage
```

Comment:
Noting to myself that this will change to:

```suggestion
            twoLineCell.leftImageView.manuallySetImage(firefoxSuggestion.iconImage)
```

After #16343 lands.